lib/thredded.rb

```diff
@@ -65,11 +66,21 @@ module Thredded
   self.user_name_column = :name
   self.content_visible_while_pending_moderation = true
   self.show_topic_followers = false
+  self.messageboards_order = :position
 
   def self.user_display_name_method
     @@user_display_name_method || user_name_column
   end
 
+  def self.messageboards_order=(value)
```

Comment:
@timdiggins I think `cattr_accessor` is an alias for `mattr_accessor` (http://api.rubyonrails.org/classes/Module.html#method-i-cattr_accessor). But yeah would love to know if there is a better way. We could also simply do `class << self; attr_acessor ... end`